tests/server/server-getServerDescription-001.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+MongoDB\Driver\Server::getServerDescription()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = create_test_manager();
+
+var_dump($manager->selectServer(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY))->getServerDescription());
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+array(%d) {
+  %a
```

Comment:
We may want to replace the `EXPECTF` block with some more detailed assertions. Due to the different formats it may be problematic to use this way of matching the returned data, but we could work around this by inspecting the resulting array in the test (as is done in other tests where we have the same restrictions.